internal/catalogmetadata/client/client_test.go

```diff
@@ -29,29 +33,34 @@ func init() {
 	utilruntime.Must(catalogd.AddToScheme(scheme))
 }
 
+// TODO(everettraven): Add unit tests for testing the caching logic.
+// Holding off on this until the caching implementation is agreed upon
+// (i.e memory based vs file based caching)
```

Comment:
TODO: remove this comment

Reply:
https://github.com/operator-framework/operator-controller/pull/411/commits/f23b319c87b1b3e87d3d1217c6488537989f0a2d